### PR TITLE
Django 2.0 compatibility

### PR DESCRIPTION
**Pull request by hebertjulio**
Fix issue #21

### CODE REVIEW

**Comment by ludrao** [comment]
Thx @hebertjulio !
I suppose that you were able to test it on both pre/post 2.0 Django versions?

File: tellme/urls.py [hunk 1]
```diff
@@ -1,13 +1,30 @@
-from django.conf.urls import url, include
-from django.views.i18n import javascript_catalog
```
Comment (ludrao) [inline tellme/urls.py:2]: thx for the cleanup ;-)

File: tellme/views.py [hunk 2]
```diff
@@ -22,7 +27,7 @@ def post_feedback(request):
                         'user': request.user.id}
         else:
             data = {'url': feedback['url'], 'browser': json.dumps(feedback['browser']), 'comment': feedback['note'],
-                        'email': feedback['email']}
+                        'email': feedback.get('email')}
```
Comment (ludrao) [inline tellme/views.py]: how is this change related ? And should it be done for the other fields of the form ?

**Comment by hebertjulio** [comment]
Yes @ludrao , I test it on both, Django 1.11.10 and 2.0.1 :)
